lib/rest/index.js

```diff
@@ -9,7 +9,11 @@ module.exports = function ({ plugins, config } = {}) {
     logger.verbose('Admin server is not configured, launch canceled');
     return;
   }
-  cfg.admin.hostname = cfg.admin.hostname || 'localhost';
+  if (cfg.admin.hostname) {
+    // eslint-disable-next-line no-console
+    console.log('Warning! use of hostname is depracated in admin section, use host instead');
```

Comment:
Do not use `console.log`, use the `logger object`

```suggestion
    logger.warn('Warning! use of hostname is depracated in admin section, use host instead');
```